apps/ledger-live-desktop/tests/page/ledgerSync.page.ts

```diff
@@ -0,0 +1,102 @@
+import { AppPage } from "tests/page/abstractClasses";
+import { step } from "tests/misc/reporters/step";
+import { expect } from "@playwright/test";
+
+export class LedgerSyncPage extends AppPage {
```

Comment:
I think this should be drawer and not page, therefore it should also inherit from Drawer class